rb/lib/selenium/webdriver/common/selenium_manager.rb

```diff
@@ -72,8 +70,8 @@ def generate_command(binary, options)
         # @return [String] the path to the correct selenium manager
         def binary
           @binary ||= begin
-            path = File.expand_path(BIN_PATH, __FILE__)
-            path << if Platform.windows?
+            path = $LOAD_PATH.find { |path| path.match(/\/bin$/) }
```

Comment:
$LOAD_PATH is one for all Ruby gems - I don't think it's going to work like that.

Reply:
I'll stop being clever and do the explicit thing.